workloads/ping-pong/ping-pong-client/deploy.yaml

```diff
@@ -26,7 +26,8 @@ spec:
       serviceAccountName: ping-pong-client
       containers:
       - name: ping-pong-client
-        image: ko://github.com/cofide/cofide-demos/workloads/ping-pong/client
+        image: ghcr.io/cofide/cofide-demos/ping-pong-client:latest
```

Comment:
This non-mesh change will have quite a wide-ranging impact.

Reply:
The main dependency beyond the ones I've started to address is over in `cofidectl`, where we clone this repo directly

https://github.com/cofide/cofidectl/blob/7fe5ab8339f5bfd571f1546a71b380c115468b20/demos/Justfile#L34-L42

I think we can remove a lot of this in `cofidectl` in favour of using the image at a specific tag. What do you think?

Reply:
I tested [this `cofidectl` diff](https://github.com/cofide/cofidectl/pull/167/files) locally with the integration tests and it seems happy enough. Letting the CI run on that draft PR to see if anything else comes out in the wash

The main yuck bit is `COFIDE_DEMOS_BRANCH` referring to this feature branch (to save on breakages), but once this is merged we can change all the references to `main` here and elsewhere. Ultimately this will be dealt with by https://github.com/cofide/cofide-demos/issues/23 but this seems a decent enough intermediate step that unblocks released container image usage

Reply:
CI is green on `cofidectl` using these images https://github.com/cofide/cofidectl/pull/167

Reply:
I actually think targeting a specific version of cofide-demos in cofidectl is a good idea, at least until we have a way to pin the deployment manifests.